cross/openzwave/Makefile

```diff
@@ -0,0 +1,30 @@
+PKG_NAME = openzwave
+PKG_VERS = $(PKG_SVN_REV)
+PKG_EXT = tar.gz
+PKG_DOWNLOAD_METHOD = svn
+PKG_DOWNLOAD_VCS_CLEAN = nope
```

Comment:
Couldn't you just download the snapshots directly? http://old.openzwave.com/snapshots/.
Btw, the snapshots are deemed unstable. If possible, stick to the stable releases: http://old.openzwave.com/downloads/